### DISCUSSION TITLE
在readme.md中添加Start History（Star 历史记录）

### DISCUSSION THREAD

**Discussion by zsr-lukezhang** [win12-online/win12#403] [Ideas]
可以在`Readme.md`中加入`Star History`模块。（即 Star 历史记录）
内容如下：

```
## Star History

[![Star History Chart](https://api.star-history.com/svg?repos=tjy-gitnub/win12&type=Date)](https://star-history.com/#tjy-gitnub/win12&Date)

```
预览：

## Star History

[![Star History Chart](https://api.star-history.com/svg?repos=tjy-gitnub/win12&type=Date)](https://star-history.com/#tjy-gitnub/win12&Date)

**Comment by lingbopro** [comment]
我打算开个PR把Star History加上，现在就开...（bushi

**Comment by zsr-lukezhang** [comment]
怎么...现在这个团队这么多人...我以前2023还是2022年年初好像只有TJY一个人啊...
顺便好奇问一下，PR是什么？
（我即GitHub@Github-Luke-Zhang，只是换了个号罢了）

**Reply by lingbopro** [reply]
就是拉取请求(Pull Request)的缩写，具体可以看[GitHub的官方文档](https://docs.github.com/zh/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)

**Reply by tjy-gitnub** [reply]
>u-)o

**Comment by lingbopro** [comment]
不对劲啊  
这个讨论咋还没关

**Comment by zsr-lukezhang** [comment]
@lingbopro 我忘了，我的错
